openlibrary/components/MergeUI/utils.js

```diff
@@ -126,14 +126,7 @@ export function make_redirect(master_key, dupe) {
 
 export function get_editions(work_key) {
     const endpoint = `${work_key}/editions.json`;
-    let base = '';
-    if (CONFIGS.OL_BASE_BOOKS) {
-        base = CONFIGS.OL_BASE_BOOKS;
-    } else {
-        // FIXME Fetch from prod openlibrary.org, otherwise it's outdated
```

Comment:
I think we'll want to keep this logic in; it's important for preventing some subtle bugs on testing.

Reply:
Why is it only on some endpoints?

Reply:
Because I believe it only affects testing.openlibrary.org or staging.openlibrary.org ; they have a separate instance of memcached which causes an annoying subtle bug resulting in testing seeing slightly outdated records. I.e. if an edit happens, testing's cache isn't cleared correctly, resulting in it having an old copy. This means if you do an edit on testing while it's in this state, it will _edit the old record_, effectively undoing the edit that was on prod.

It's a headache of a bug :P